src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -301,8 +301,14 @@ public with sharing class CRLP_RollupUI_SVC {
         Set<String> excludedTypes = new Set<String>{'TIME', 'TEXTAREA'};
 
         for(String obj : objectNames){
-            List<Map<String, String>> detailNameToFieldTypeObj = getFilterRuleFieldsExcludingTypes(obj, excludedTypes);
-            objNameToFieldTypeMap.put(obj, detailNameToFieldTypeObj);
+            if (obj == 'Partial_Soft_Credit__c') {
+                List<String> fieldsToReturn = new List<String>{'Role_Name__c','Amount__c'};
```

Comment:
These field names won't work this way either in the final package. Use the `UTIL_Namespace.StrAllNSPrefix()` method to apply the namespace as needed.